hooks/shell.hook.js

```diff
@@ -20,29 +20,42 @@ class hook extends baseWidget() {
   }
 
   openShell () {
-    const dockerRunScriptPath = `${__dirname}/../dockerRunScript.sh`
-    let containerId = this.widgetsRepo.get('containerList').getSelectedContainer()
+    const containerList = this.widgetsRepo.get('containerList')
+    const containerId = containerList.getSelectedContainer()
 
     if (containerId) {
-      let containerIdFile = path.join(__dirname, '/../containerId.txt')
-      fs.writeFile(containerIdFile, containerId, 'utf8', (err) => {
-        if (!err) {
-          TerminalLauncher.launchTerminal({ path: dockerRunScriptPath }).catch((err) => {
-            console.log(err)
-            const actionStatus = this.widgetsRepo.get('actionStatus')
-
-            const title = 'Shell login to container'
-            const message = 'Failed opening shell login for container: ' + containerId + ' - ' + err
-
-            actionStatus.emit('message', {
-              title: title,
-              message: message
-            })
+      this.utilsRepo.get('docker').getContainer(containerId, (_, container) => {
+        if (!container.State.Running) {
+          this.emitActionStatus('Oops!', 'Cannot attach interactive shell - Container is not running.')
+          return
+        }
+
+        try {
+          console.clear()
+
+          cp.execSync(`docker exec -it "${containerId}" /bin/sh`, {
```

Comment:
It's a small nit, but valuable :)
Could you perhaps replace `cp.execSync` with `cp.execFileSync` ?